bwresources.py

```diff
@@ -204,6 +204,10 @@ def num_mentions(self, **kwargs):
         params = self._fill_mentions_params(kwargs)
         return self.project.get(endpoint="data/mentions/count", params=params)
 
+    def get_chart(self, y_axis, x_axis, breakdown_by, **kwargs):
```

Comment:
Would it be possible to briefly document possible choices for `y_axis`, `x_axis`, and `breakdown_by`?  (Even if it's just a reference to `filters.py` or something.)